client/components/data/query-verticals/index.jsx

```diff
@@ -17,28 +18,48 @@ export class QueryVerticals extends Component {
 	static propTypes = {
 		searchTerm: PropTypes.string,
 		limit: PropTypes.number,
+		debounceTime: PropTypes.number,
+		debounceFunc: PropTypes.func,
```

Comment:
Should we add `isFetched` to the propTypes here?

Also, check the value in `connect`. We should be passing `state` as the first arg, no?

`isFetched: null !== getVerticals( state, ownProps.searchTerm ),`